spec/03-plugins/23-aws-lambda/01-access_spec.lua

```diff
@@ -499,4 +518,34 @@ describe("Plugin: AWS Lambda (access)", function()
     assert.res_status(412, res)
     assert.equal("Unhandled", res.headers["X-Amz-Function-Error"])
   end)
+  it("invokes a Lambda function with an handled function error", function()
+    local res = assert(client:send {
+      method = "POST",
+      path = "/post",
+      headers = {
+        ["Host"] = "lambda12.com",
+        ["Content-Type"] = "application/x-www-form-urlencoded"
+      },
+      body = {
+        key1 = "Error:400"
+      }
+    })
+    assert.res_status(400, res)
+    assert.equal("Handled", res.headers["X-Amz-Function-Error"])
+  end)
+  it("invokes a Lambda function with an handled function error", function()
```

Comment:
This test name should be different to the previous one and should indicate that this is a success case, rather than an error case.